src/System.IO.MemoryMappedFiles/src/Interop/Interop.Windows.cs

```diff
@@ -211,6 +210,12 @@ internal static partial class mincore
             ref SECURITY_ATTRIBUTES lpFileMappingAttributes, int flProtect,
             int dwMaximumSizeHigh, int dwMaximumSizeLow, string lpName);
 
+        [DllImport(MEMORYDLL, EntryPoint = "CreateFileMappingW", CharSet = CharSet.Unicode, SetLastError = true)]
+        [SecurityCritical]
+        internal static extern SafeMemoryMappedFileHandle CreateFileMapping(IntPtr hFile,
```

Comment:
I'd added this overload so that it wouldn't need to be a SafeHandle, since the value will always be -1. I could make this overload private and add an internal wrapper that doesn't expose an hFile parameter and always calls it with -1... ? The alternative I considered was to cache a SafeFileHandle instance for -1 and use that, just seemed unnecessary.